packages/wallets/src/components/OptionsAndMultipliersListing/OptionsAndMultipliersListing.scss

```diff
@@ -47,14 +48,20 @@
         grid-template-columns: repeat(3, 1fr);
         user-select: none;
 
+        @include mobile {
+            grid-template-rows: repeat(5, 1fr);
+            grid-template-columns: auto;
+            gap: 0;
+        }
+
         &__icon {
             cursor: pointer;
         }
 
-        @include mobile {
-            margin: auto 2rem;
-            grid-template-rows: repeat(5, 1fr);
-            grid-template-columns: auto;
+        &-no-trading-account {
```

Comment:
is this element or modifier?

Reply:
@wojciech-deriv suppose it's a modifier :) As I know element must be named from `__` and I thought modifiers must be with `--`, but I wrote just one `-`. I'll change it soon